### ISSUE TITLE
Tutorial Docs: Second tutorial Increment struct mismatch

### ISSUE THREAD

**Issue by Ahmed-Ali** [project-serum/anchor#787]
Hi,
Big fan here :) Anchor indeed elevate the ease of use Solana and makes things much more convenient! thank you!

Back to the issue. In the [second tutorial](https://github.com/project-serum/anchor/blob/master/docs/src/tutorials/tutorial-2.md) I found the document referencing the Increment struct as
```
#[derive(Accounts)]
pub struct Increment<'info> {
    #[account(mut, has_one = authority)]
    pub counter: Account<'info, Counter>,
    #[account(signer)]
    pub authority: AccountInfo<'info>,
}
```

However, in the current code it is
```
#[derive(Accounts)]
pub struct Increment<'info> {
    #[account(mut, has_one = authority)]
    pub counter: Account<'info, Counter>,
    pub authority: Signer<'info>,
}
```

So seems like this tutorial needs a revise to update its information.

**Comment by nicholasoxford** [comment]
I opened #796 to address this..

Also add me on discord `nick-atl#0745`, between Anchor and Realm it seems we have a lot of similar interest!

**Comment by armaniferrante** [comment]
Closed by #796.
